src/react/components/searchResults.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import sanitize from "sanitize-html";
+import HeroIcon from './heroIcon'
+
+const SearchResults = ({ results }) => {
+  if (!results.hits) {
+    return <div />;
+  }
+
+  return (
+    <div id="search-results" className="su-sans">
+      <div className="algolia-search--results-count lg:mb-16">
+        <span className="font-semibold">{results.nbHits}</span> results:
+      </div>
+      {results.hits.map((result) => (
+        <div
+          key={result.objectID}
+          className="algolia-search--result-item px-0 py-16 md:px-16 border-b"
+        >
+          <div className="flex flex-wrap md:flex-nowrap">
+            <div className="md:flex-1 w-full">
+              <div className="algolia-search--result-domain mb-4">{result.domain}</div>
+              <h3 className="algolia-search--result-title mb-6 su-serif font-bold">
+                <a
+                  className="transition-colors hover:underline focus:underline"
+                  href={result.url}
+                >
+                  {result.fileType === "video" && (
+                    <HeroIcon
+                      iconType="video"
+                      className="inline-block mr-2"
+                      srText="Video: "
+                    />
+                  )}
+                  {result.fileType === "audio" && (
+                    <HeroIcon
+                      iconType="podcast"
+                      className="inline-block -ml-1"
+                      srText="Podcast: "
+                    />
+                  )}
+                  <span
+                    // eslint-disable-next-line react/no-danger
+                    dangerouslySetInnerHTML={{
+                      // eslint-disable-next-line no-underscore-dangle
+                      __html: sanitize(result.title, { decodeEntities: false }),
+                    }}
+                  />
+                  <HeroIcon
```

Comment:
I think we need more spacing between the trailing icons and the text in the search results heading. Please have it match the homesite spacing.